src/aux-runtime/runtime/AuxLibrary.ts

```diff
@@ -6418,6 +6422,32 @@ export function createDefaultLibrary(context: AuxGlobalContext) {
         return addAsyncAction(task, action);
     }
 
+    /**
+     * Classifies the given images using the image classifier. Returns a promise that resolves with the results of the classification.
+     *
+     * @param options the options that should be used for the image classification.
+     *
+     * @example Classify the given images.
+     * const files = await os.showUploadFiles()
+     * const classify = os.classifyImages({
+     *      modelUrl: "MY_MODEL_URL",
+     *      images: files.map((file) => {
+     *         return {file}
+     *      })
+     * })
+     *
+     * @dochash actions/os/image-classification
+     * @docname os.classifyImages
+     * @docgroup 10-image-classifier
+     */
+    function classifyImages(
+        options: ClassifyImagesOptions
+    ): Promise<ClassifyImagesResult> {
+        const task = context.createTask();
+        const action = calcOpenClassifyImages(true, options, task.taskId);
```

Comment:
@TroyceGowdy `true` doesn't need to be specified because the `open` property should be removed.